user_management/tests/run.py

```diff
@@ -76,7 +85,7 @@ class TestRunner(ColourRunnerMixin, DiscoverRunner):
     pass
 
 
-test_runner = TestRunner(verbosity=1)
+test_runner = TestRunner(verbosity=1, keepdb=True)
```

Comment:
I can imagine this is very helpful... until it isn't, and needs to be changed in the file by hand.

This might be worth getting from `sys.argv`.

Reply:
I usually just use psql to drop the db, but an arg is a good idea.